test/acceptance/test/add_tests.go

```diff
@@ -398,16 +401,17 @@ var _ = Describe("Weave GitOps Add Tests", func() {
 		})
 
 		By("And I run wego add command with specified branch", func() {
-			addCommandOutput, _ = runWegoAddCommandWithOutput(repoAbsolutePath, addCommand, WEGO_DEFAULT_NAMESPACE)
+			runWegoAddCommand(repoAbsolutePath, addCommand, wegoNamespace)
 		})
 
-		By("Then I should see wego using the specified url", func() {
-			Eventually(addCommandOutput).Should(ContainSubstring("branch: " + branchName))
+		By("And I should see my workload deployed to the cluster", func() {
+			verifyWegoAddCommand(appName, wegoNamespace)
+			verifyWorkloadIsDeployed("nginx", "my-nginx")
 		})
 
-		By("And I should see should see my workload deployed to the cluster", func() {
-			verifyWegoAddCommand(appName, WEGO_DEFAULT_NAMESPACE)
-			verifyWorkloadIsDeployed("nginx", "my-nginx")
+		By("And I verify my specified branch name", func() {
+			branchOutput := checkGitBranch(repoAbsolutePath)
+			Eventually(branchOutput).Should(ContainSubstring(branchName))
 		})
```

Comment:
Is this verifying the local branch? Wondering if the test can verify the branch on the cluster?

Reply:
Not sure what you mean, but it's actually verifying the branch that I created and used during the `wego app add`. I wanted to make sure I added the app in my specific branch (test-branch).